### PR TITLE
Add sizes to models

### PR DESCRIPTION
**Pull request by NikitaDenysenko**
The models can now accept array of sizes, which are related to them

### CODE REVIEW

File: modules/model/model.graphql.js [hunk 1]
```diff
@@ -7,11 +7,15 @@ type Model {
     images: ImageSet,
     priority: Int,
     show: Boolean,
+<<<<<<< HEAD
```
Comment (moran711) [inline modules/model/model.graphql.js:10]: Please fix this merge conflict.

File: modules/model/model.graphql.js [hunk 2]
```diff
@@ -23,10 +27,14 @@ input ModelInput {
     images: ImageSetInput,
     priority: Int,
     show: Boolean,
+<<<<<<< HEAD
```
Comment (moran711) [inline modules/model/model.graphql.js:30]: Please fix this merge conflict.

**Comment by ivan-bonk** [comment]
Fix duplications and code smell please
Approve after fixes

File: modules/model/model.graphql.js (hunk 1 above)
Comment (NikitaDenysenko) [reply modules/model/model.graphql.js:10]: done

File: modules/model/model.graphql.js (hunk 2 above)
Comment (NikitaDenysenko) [reply modules/model/model.graphql.js:30]: done

File: modules/model/model.model.js [hunk 3]
```diff
@@ -12,11 +13,25 @@ const modelSchema = new mongoose.Schema({
...
+    },
+  ],
   availableForConstructor: Boolean,
-  constructorBasic: [{ type:mongoose.Schema.Types.ObjectId, ref: 'ConstructorBasic'}],
-  constructorPattern: [{ type:mongoose.Schema.Types.ObjectId, ref: 'Pattern'}],
-  constructorFrontPocket: [{ type:mongoose.Schema.Types.ObjectId, ref: 'ConstructorFrontPocket'}],
-  constructorBottom: [{ type:mongoose.Schema.Types.ObjectId, ref: 'ConstructorBottom'}],
+  constructorBasic: [
+    { type: mongoose.Schema.Types.ObjectId, ref: 'ConstructorBasic' },
+  ],
```
Comment (kapoliub) [inline modules/model/model.model.js:25]: change it for 'constructorBasics'

File: resolvers.js [hunk 4]
```diff
@@ -206,7 +206,11 @@ const resolvers = {
   },
 
   Model: {
-    category: parent => categoryService.getCategoryById(parent.category)
+    category: parent => categoryService.getCategoryById(parent.category),
+    sizes: parent =>
+      parent.sizes.map((size, index) =>
+        sizeService.getSizeById(parent.sizes[index])
+      ),
```
Comment (kapoliub) [inline resolvers.js:213]: why you use as parameter `parent.sizes[index]`. You can just use `size` from your `map` arguments

Comment (NikitaDenysenko) [reply resolvers.js]: done

**Review by kapoliub — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

File: modules/size/size.resolver.js [hunk 5]
```diff
@@ -16,9 +16,9 @@ const sizeQuery = {
 };
 
 const sizeMutation = {
-  addSize: async (parent_, { data }) => {
+  addSize: async (parent_, { size }) => {
```
Comment (moran711) [inline modules/size/size.resolver.js:19]: Please rename `parent_` to `parent`

**Review by moran711 — APPROVED** [review]

File: modules/size/size.resolver.js (hunk 5 above)
Comment (NikitaDenysenko) [reply modules/size/size.resolver.js]: done

**Review by ivan-bonk — APPROVED** [review]

**Pull request merged**